css-view-transitions-2/Overview.bs

```diff
@@ -1182,7 +1182,7 @@ When capturing the old or new state for an element, perform the following steps
 
 To get the <dfn>nearest containing group name</dfn> for an {{Element}} |element|, perform the following steps:
 
-	1. Let |nearestAncestorWithContain| be |element|'s nearest ancestor whose 'view-transition-name' [=computed value=] is not ''view-transition-name/none'' and whose 'view-transition-group' [=computed value=] is ''view-transition-group/contain''.
+	1. Let |nearestAncestorWithContain| be |element|'s nearest [=tree/ancestor=] whose 'view-transition-name' [=computed value=] is not ''view-transition-name/none'' and whose 'view-transition-group' [=computed value=] is not ''view-transition-group/normal''.
```

Comment:
Just confirming that there's no edge case here. There are cases where we ignore element's with a name here: https://drafts.csswg.org/css-view-transitions-1/#capture-the-old-state. It's in step 8 for elements without a box or if it's c-v: hidden/auto. In all such cases the element's descendants will also be ignored so it doesn't really matter.

If that sounds good, maybe leave a note.

Reply:
See refactor